src/webui/src/components/trial-detail/TableList.tsx

```diff
@@ -45,749 +39,497 @@ echarts.registerTheme('my_theme', {
-        // parameter as table column
-        const parameterStr: string[] = [];
-        if (!EXPERIMENT.isNestedExp()) {
-            if (tableSource.length > 0) {
-                const trialMess = TRIALS.getTrial(tableSource[0].id);
-                const trial = trialMess.description.parameters;
-                const parameterColumn: string[] = Object.keys(trial);
-                parameterColumn.forEach(value => {
-                    parameterStr.push(`${value} (search space)`);
-                });
+    private _buildColumnsFromTableItems(tableItems: any[]): IColumn[] {
+        // extra column, for a icon to expand the trial details panel
+        const columns: IColumn[] = [
+            {
+                key: '_expand',
+                name: '',
+                onRender: (item, index): any => {
+                    return (
+                        <Icon
+                            aria-hidden={true}
+                            iconName='ChevronRight'
+                            styles={{
+                                root: {
+                                    transition: 'all 0.2s',
+                                    transform: `rotate(${item._expandDetails ? 90 : 0}deg)`
+                                }
+                            }}
+                            onClick={(event): void => {
+                                event.stopPropagation();
+                                const newItem: any = { ...item, _expandDetails: !item._expandDetails };
+                                const newItems = [...this.state.displayedItems];
+                                newItems[index as number] = newItem;
+                                this.setState({
+                                    displayedItems: newItems
+                                });
+                            }}
+                            onMouseDown={(e): void => {
+                                e.stopPropagation();
+                            }}
+                            onMouseUp={(e): void => {
+                                e.stopPropagation();
+                            }}
+                        />
+                    );
+                },
+                fieldName: 'expand',
+                isResizable: false,
+                minWidth: 20,
+                maxWidth: 20
             }
+        ];
+        // looking at the first row only for now
+        for (const k of Object.keys(tableItems[0])) {
+            const lengths = tableItems.map(item => `${item[k]}`.length);
+            const avgLengths = lengths.reduce((a, b) => a + b) / lengths.length;
+            const columnTitle = _inferColumnTitle(k);
+            const columnWidth = Math.max(columnTitle.length, avgLengths);
+            // TODO: add blacklist
+            columns.push({
+                name: columnTitle,
+                key: k,
+                fieldName: k,
+                minWidth: columnWidth * 4,
+                maxWidth: columnWidth * 10,
```

Comment:
you can try this size: 
                minWidth: columnWidth * 13,
                maxWidth: columnWidth * 18,